hack/build-s2i-dpdk.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+ELAPSED=0
+TIMEOUT=180
+ATTEMPTS=0
+MAX_ATTEMPTS=10
+build_complete=false
+
+until $build_complete || [ $ELAPSED -eq $TIMEOUT ] || [ $ATTEMPTS -eq $MAX_ATTEMPTS ]
+do
+    start_build=false
+    last_build=$(oc get build -n dpdk -o json | jq '.items[-1].metadata.name' | tr -d '"') 
+    if [ $last_build == "null" ]; then
+        start_build=true
+    else
+        build_status=$(oc get build -n dpdk $last_build -o json | jq '.status.phase' | tr -d '"')
+        if [ $build_status == "Complete" ]; then
+            build_complete=true
+        elif [ $build_status == "Failed" ]; then
+            start_build=true
+        fi
+    fi
+    if $start_build; then
+        oc start-build -n dpdk s2i-dpdk
+        (( ATTEMPTS++ ))
+    fi
+    sleep 10
+    (( ELAPSED++ ))
```

Comment:
sleep 10
then add just 1 to ELAPSED?
so timeout 180 actually means 1800 seconds sound like wayy too much

I think 30 minutes overall should be more than enough for test_pmd on any HW

Reply:
1800 seconds = 30 minutes